### ISSUE TITLE
Look for Pretrained Model to Throw at Frames Adding in Panda Labels

### ISSUE THREAD

**Issue by jzf2101** [rubyforgood/rp_ml#8]
See http://cs231n.github.io/transfer-learning/ and tensorflow/issues/6
